Issue with kali-rolling

```
Please paste the output of the following command here: sudo edit-chroot -all

name: kali-rolling
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/kali-rolling...
crouton: version 1-20190807171712~master:6f45f06a
release: kali-rolling
architecture: amd64
targets: x11,xorg,audio,gtk-extra,gnome
host: version 12404.0.0 (Official Build) canary-channel eve 
kernel: Linux localhost 4.4.186-16724-g284c626f060d #1 SMP PREEMPT Wed Aug 7 21:07:31 PDT 2019 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...
```

#### Please describe your issue:
Error with buffer length in strncpy when installing kali-rolling

```
common/cras_shm.c: In function 'cras_shm_info_init':
common/cras_shm.c:29:2: error: 'strncpy' specified bound 255 equals destination size [-Werror=stringop-truncation]
  strncpy(info.name, stream_name, sizeof(info.name));
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...
```


#### If known, describe the steps to reproduce the issue:
set up kali with the command `sudo crouton -t x11,xorg,audio,gtk-extra,gnome -r kali-rolling`

I'm having the same issue with both updating a Debian Bullseye and installing a Debian Sid chroot.

I'm on beta channel, Version 77.0.3865.35, running on an Asus C302.

Output of `sudo edit-chroot -all`
```name: sid
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/sid...
WARNING: Unable to register core dump handler.
crouton: version 1-20190807171712~master:6f45f06a
release: sid
architecture: amd64
targets: audio,extension,x11,xorg
host: version 12371.22.0 (Official Build) beta-channel cave 
kernel: Linux localhost 3.18.0-19728-g489554e8d014 #1 SMP PREEMPT Wed Aug 14 01:04:21 PDT 2019 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/sid...```

Output of updating Bullseye with targets; audio,xiwi,extension,x11.
```Compiling CRAS (i386)...
common/cras_shm.c: In function 'cras_shm_info_init':
common/cras_shm.c:29:2: error: 'strncpy' specified bound 255 equals destination size [-Werror=stringop-truncation]
  strncpy(info.name, stream_name, sizeof(info.name));
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/bullseye...
Sending SIGTERM to processes under /mnt/stateful_partition/crouton/chroots/bullseye...```

Output of installing Debian Sid with targets; audio,extension,x11,xorg.
```Compiling CRAS (i386)...
common/cras_shm.c: In function 'cras_shm_info_init':
common/cras_shm.c:29:2: error: 'strncpy' specified bound 255 equals destination size [-Werror=stringop-truncation]
   29 |  strncpy(info.name, stream_name, sizeof(info.name));
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/sid...```

--- a/targets/audio
+++ b/targets/audio
@@ -157,6 +157,10 @@ END
 	sed -i -e 's/cras_stream_id_t stream_id;/cras_stream_id_t stream_id = 0;/' \
 	    "$cras_test_client"
 
+	# FIXME(http://crrev.com/c/1759370): Remove when fixed upstream
+	sed -i -e 's/\(strncpy.*sizeof(info.name)\));/\1 - 1); info.name[sizeof(info.name) - 1] = '"'"'\\0'"'"';/' \
+		common/cras_shm.c
+
         # Directory to install CRAS library/binaries
         CRASLIBDIR="/usr/local$archextrapath/lib"
         CRASBINDIR="/usr/local$archextrapath/bin"

